gusto/diagnostics.py

```diff
@@ -92,8 +140,72 @@ def field(self, mesh):
 
     def compute(self, state):
         u = state.fields("u")
-        uh = u[0]
-        return self.field(state.mesh).interpolate(uh)
+        v = u[1]
+        return self.field(state.mesh).interpolate(v)
```

Comment:
This is what I meant. I would like to track v in Eady tests which uses a 3D mesh.